src/lib/eventManager.js

```diff
@@ -14,6 +14,17 @@ export function setupEventListeners(root) {
   // 2. 기존에 설정된 이벤트 리스너 제거 (있다면)
   // 3. eventMap에 등록된 모든 이벤트 타입에 대해 루트 요소에 이벤트 리스너 추가
   // 주의: 이벤트 캡처링을 사용하여 이벤트를 상위에서 하위로 전파
+
+  rootElement = root;
+
+  Array.from(eventMap.keys()).forEach((event) => {
+
+    // removeEvent(rootElement, event, handleEvent)
+    rootElement.removeEventListener(event, handleEvent, true);
+    rootElement.addEventListener(event, handleEvent, true)
+  })
```

Comment:
```suggestion
 eventMap.forEach((_,event) => {

    // removeEvent(rootElement, event, handleEvent)
    rootElement.removeEventListener(event, handleEvent, true);
    rootElement.addEventListener(event, handleEvent, true)
  })
```

Map에도 forEach 메소드가 있어서, Array로 변환하지 않아도 forEach를 사용할 수 있을 것 같아용 ㅎㅎ

아래는 Map에 대해 내장 인터페이스로 등록된 부분입니당! 😆
<img width="838" alt="image" src="https://github.com/user-attachments/assets/5a8c9c8f-2caa-4211-9af5-edb782f650ae">